<launch>

  <node name="image_to_octree" type="image_to_octree" pkg="nbv_3d_cnn_octree" output="screen" clear_params="true" >
    <param name="source_prefix" type="string" value="$(find nbv_3d_cnn)/data/environments_3d_realistic/" />
    <!--<param name="source_prefix" type="string" value="$(find nbv_3d_cnn_octree)/data/environments_3d_realistic2x/environments_3d_realistic2x/" />-->

    <param name="is_3d" type="bool" value="true" />

    <param name="dest_prefix" type="string" value="$(find nbv_3d_cnn_octree)/data/environments_3d_realistic/" />

    <param name="max_levels" type="int" value="6" />
  </node>

</launch>
